docs/translate-usage.rst

```diff
@@ -0,0 +1,109 @@
+Using the API
+=============
+
+With `Google Translate`_, you can dynamically translate text
+between thousands of language pairs. The Google Translate API
+lets websites and programs integrate with Google Translate
+programmatically. Google Translate API is available as a
+paid service. See the `Pricing`_ and `FAQ`_ pages for details.
+
+Authentication / Configuration
+------------------------------
+
+- Use :class:`~gcloud.translate.client.Client` objects to configure
+  your applications.
+
+- :class:`~gcloud.translate.client.Client` objects hold both a ``key``
+  and a connection to the Translate service.
+
+- **An API key is required for Translate.** See
+  `Identifying your application to Google`_ for details. This is
+  significantly different than the other clients in ``gcloud-python``.
+
+Methods
+-------
+
+To create a client:
+
+  .. code::
+
+     >>> from gcloud import translate
+     >>> client = translate.Client('my-api-key')
+
+The API has three supported methods:
+:meth:`~gcloud.translate.client.Client.get_languages`,
+:meth:`~gcloud.translate.client.Client.detect_language` and
+:meth:`~gcloud.translate.client.Client.translate`.
+
+To get a list of languages supported by Google Translate
+
+  .. code::
+
+     >>> from gcloud import translate
+     >>> client = translate.Client('my-api-key')
+     >>> client.get_languages('en')
+     [
+         {
+             'language': 'af',
+             'name': 'Afrikaans',
+         },
+          ...
+     ]
+
+To detect the language that some given text is written in:
+
+  .. code::
+
+     >>> from gcloud import translate
+     >>> client = translate.Client('my-api-key')
+     >>> client.detect_language(['Me llamo', 'I am'])
+     [
+         {
+             'confidence': 0.25830904,
+             'input': 'Me llamo',
+             'language': 'es',
+         }, {
+             'confidence': 0.17112699,
+             'input': 'I am',
+             'language': 'en',
+         },
+     ]
+
+To translate text:
+
+  .. code::
+
+     >>> from gcloud import translate
+     >>> client = translate.Client('my-api-key')
+     >>> client.translate('koszula')
+     [
+         {
+             'translatedText': 'shirt',
+             'detectedSourceLanguage': 'pl',
+             'input': 'koszula',
+         },
+     ]
+
+or to use a non-default target language:
```

Comment:
Do we have an example of creating a client with another default? is this what it looks like?

```python
client = translate.Client('api-key', default_target='es')
```